pkg/wasm/convert.go

```diff
@@ -37,26 +37,33 @@ import (
 	"istio.io/istio/pkg/util/protomarshal"
 )
 
+const (
+	// DefaultAllowStatPrefix is the stat prefix used for WASM default allow RBAC filters
+	DefaultAllowStatPrefix = "wasm-default-allow"
```

Comment:
Can we use the same constants in the test code - https://github.com/istio/istio/blob/master/pkg/wasm/convert_test.go